src/game/common/rts/team.cpp

```diff
@@ -0,0 +1,817 @@
+    }
+
+    m_scriptOnCreate = d->Get_AsciiString(g_teamOnCreateScriptKey, &exists);
+    m_isAIRecruitable = d->Get_Bool(g_teamIsAIRecruitableKey, &exists);
+
+    if (!exists) {
+        m_isAIRecruitable = false;
+    }
+
+    m_isBaseDefense = d->Get_Bool(g_teamIsBaseDefenseKey, &exists);
+    m_isPerimeterDefense = d->Get_Bool(g_teamIsPerimeterDefenseKey, &exists);
+    m_automaticallyReinforce = d->Get_Bool(g_teamAutoReinforceKey, &exists);
+    int aggressiveness = d->Get_Bool(g_teamAggressivenessKey, &exists);
+    m_aggressiveness = 0;
+
+    if (exists) {
+        m_aggressiveness = aggressiveness;
+    }
+
+    m_transportsReturn = d->Get_Bool(g_teamTransportsReturnKey, &exists);
+    m_avoidThreats = d->Get_Bool(g_teamAvoidThreatsKey, &exists);
+    m_attackCommonTarget = d->Get_Bool(g_teamAttackCommonTargetKey, &exists);
+    m_maxInstances = d->Get_Int(g_teamMaxInstancesKey, &exists);
+    m_scriptOnIdle = d->Get_AsciiString(g_teamOnIdleScriptKey, &exists);
+    m_initialIdleFrames = d->Get_Int(g_teamInitialIdleFramesKey, &exists);
+    m_scriptOnEnemySighted = d->Get_AsciiString(g_teamEnemySightedScriptKey, &exists);
+    m_scriptOnAllClear = d->Get_AsciiString(g_teamAllClearScriptKey, &exists);
+    m_scriptOnDestroyed = d->Get_AsciiString(g_teamOnDestroyedScriptKey, &exists);
+    m_destroyedThreshold = d->Get_Real(g_teamDestroyedThresholdKey, &exists);
+    m_scriptOnUnitDestroyed = d->Get_AsciiString(g_teamOnUnitDestroyedScriptKey, &exists);
+    m_productionPriority = d->Get_Int(g_teamProductionPriorityKey, &exists);
+    m_productionPrioritySuccessIncrease = d->Get_Int(g_teamProductionPrioritySuccessIncreaseKey, &exists);
+    m_productionPriorityFailureDecrease = d->Get_Int(g_teamProductionPriorityFailureDecreaseKey, &exists);
+    m_productionCondition = d->Get_AsciiString(g_teamProductionConditionKey, &exists);
+    m_executesActionsOnCreate = d->Get_Bool(g_teamExecutesActionsOnCreateKey, &exists);
+
+    for (int key = 0; key < ARRAY_SIZE(m_genericScripts); key++) {
+        Utf8String str;
+        str.Format("%s%d", g_theNameKeyGenerator->Key_To_Name(g_teamGenericScriptHookKey).Str(), key);
+        m_genericScripts[key] = d->Get_AsciiString(Name_To_Key(str), &exists);
+
+        if (!exists) {
+            m_genericScripts[key].Clear();
+        }
+    }
+
+    m_transport = d->Get_AsciiString(g_teamTransportKey, &exists);
+    m_transportsExit = d->Get_Bool(g_teamTransportsExitKey, &exists);
+    m_teamStartsFull = d->Get_Bool(g_teamStartsFullKey, &exists);
+    m_reifnrocementOrigin = d->Get_AsciiString(g_teamReinforcementOriginKey, &exists);
+    m_veterancy = d->Get_Int(g_teamVeterancyKey, &exists);
+}
+
+void TeamTemplateInfo::Xfer_Snapshot(Xfer *xfer)
+{
+    uint8_t version = 1;
+    xfer->xferVersion(&version, 1);
+    xfer->xferInt(&m_productionPriority);
+}
+
+TeamPrototype::TeamPrototype(
+    TeamFactory *tf, const Utf8String &name, Player *owner_player, bool is_singleton, Dict *d, unsigned int id) :
```

Comment:
owning_player

Reply:
You marked this as resolved without comment. Let me elaborate more. Currently it says:
```c++
m_owningPlayer(owner_player),
```
Is this intended choice of words?

I would have expected same names.

Reply:
Original calls it m_owningPlayer and owner_player.